virttest/utils_kernel_module.py

```diff
@@ -0,0 +1,111 @@
+import os
+import logging
+
+from avocado.utils import process
+from avocado.core import exceptions
+
+"""Handle Linux kernel modules"""
+
+
+class KernelModuleHandler(object):
+    """Class handling Linux kernel modules"""
+
+    def __init__(self, module_name):
+        self._module_name = module_name
+        self._was_loaded = True
```

Comment:
Well it would be clearer to set this to `None` to indicate it will be detected later in code.

Reply:
... and obviously set it to True/False there.